app/src/ui/lib/bytes.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Manipulate bytes for displaying purposes.
+ */
+export class Bytes {
+  public readonly bytes: number
+
+  public constructor(bytes: number) {
+    this.bytes = bytes
+  }
+
+  /**
+   * Display bytes in human readable format like:
+   * - 23gb
+   * - 43bytes
+   * @param {boolean} forceSign Show `+123kb` instead of `123kb`
+   * @return {string} formatted string
+   */
+  public format(forceSign: boolean = false) {
+    if (!Number.isFinite(this.bytes)) {
+      return 'Unknown'
+    }
+    const sizes = ['bytes', 'kb', 'mb', 'gb', 'tb']
```

Comment:
The unit for a byte is **B**, so can we update to `['B', 'KB', 'MB', 'GB', 'TB']`?

Reply:
@iAmWillShepherd I think I followed the initial design to strictly, updated, added also `PB`.